inference-engine/tests_deprecated/functional/vpu/common/layers/myriad_layers_gather_test.hpp

```diff
@@ -1,310 +0,0 @@
-// Copyright (C) 2018-2020 Intel Corporation
```

Comment:
Do we need to remove this deprecated test?

Reply:
I guess so. I migrated tests for Gather from old infra to new one, so no reason to have it there. If you see some issues with that, like there is something from old tests that was not migrated correctly to new one - please let me know.

In general, I see it as an iterative process, we migrate tests from old infra to new one to leave only new infra eventually.